SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -181,10 +184,9 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
     }
     
     @IBAction func muteSwitchToggled(_ sender: UISwitch) {
-        if let call = call {
+        if (!self.activeCalls!.isEmpty) {
+            let call = self.activeCalls![0]
```

Comment:
If a user has multiple active calls, self.activeCalls[0] may not necessarily be a live call where the user is sending and receiving media. Same applies to user initiated `disconnect()` method..